Rx.NET/Documentation/Rx-Roadmap-2023.md

```diff
@@ -0,0 +1,167 @@
+# Rx Roadmap for 2023
+
+As of February 2023, `System.Reactive` (aka Rx.NET, Rx for short in this document) has become moribund. This document describes the plan for bringing it back to life.
+
+## Current problems
+
+There are five main problems we want to address. First, there have been no recent releases. Second, the build has fallen behind current tooling. Third, Rx causes unnecessary bloat if you use it in conjunction with certain modern build techniques such as self-contained deployments and ahead-of-time compilation. Fourth, the backlog of issues has been neglected. Fifth, the asynchronous Rx code in this repo is in an experimental state, and has never been released in any form despite demand.
```

Comment:
I might be tempted to bullet these out, rather than be a dense block of text, as this is the "TL;DR" part - if you read nothing else, understand that these are what we are intending to fix in the first instance.

Reply:
I came here to say this too.

Reply:
Done